### PR TITLE
qa/rgw: remove default_idle_timeout

### PR DESCRIPTION
**Pull request by cbodley**
removed as part of fastcgi cleanup

continuation of https://github.com/ceph/ceph/pull/15400

### CODE REVIEW

File: qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml [hunk 1]
```diff
@@ -4,7 +4,6 @@ meta:
 tasks:
 - rgw: 
     client.0:
```
Comment (yehudasa) [inline qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml]: maybe `client.0: null`?

**Comment by cbodley** [comment]
scheduled upgrade and rados:upgrade suites with `--filter rgw-swift.yaml`:
http://pulpito.ceph.com/cbodley-2017-06-01_16:23:57-upgrade-master---basic-smithi
http://pulpito.ceph.com/cbodley-2017-06-01_16:25:09-rados:upgrade-master---basic-smithi

File: qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml (hunk 1 above)
Comment (cbodley) [inline qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml]: wasn't sure about that myself, is it required?

Comment (yehudasa) [inline qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml]: not 100% sure it's required, but I can't find an example where we don't put a 'null' string when dealing with empty value.

Comment (cbodley) [inline qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml]: updated

File: qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml [hunk 2]
```diff
@@ -3,8 +3,7 @@ meta:
    swift api tests for rgw
 tasks:
 - rgw: 
-    client.0:
-    default_idle_timeout: 300
+    client.0: null
```
Comment (tchaikov) [inline qa/suites/rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml]: it's safe to put 
```yaml
- rgw:
    client.0:
```

the empty value will be interpreted as an empty value, i.e. `null`, see http://www.yaml.org/spec/1.2/spec.html#e-scalar. and https://github.com/ceph/ceph/blob/master/qa/tasks/rgw.py#L186 also documents this usage.

**Comment by liewegas** [comment]
oops merged yuri's version of this, https://github.com/ceph/ceph/pull/15420

**Pull request closed without merging**